### PR TITLE
Todo type script

### PR DESCRIPTION
**Pull request by StanCosmin28**

### CODE REVIEW

File: src/components/TodoContext.tsx [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+
+  const TODO_API_URL: string = "https://jsonplaceholder.typicode.com/todos/";
+  const TODOS_NUMBER: number = 5;
+  const TODO_API_URL_LIMIT: string = `https://jsonplaceholder.typicode.com/todos?_limit=${TODOS_NUMBER}`;
+
+  const updateLocalStorage = (todos: Todo[]) => {
+    localStorage.setItem("todos", JSON.stringify(todos));
+  };
+  const { mutate: onAdd } = useMutation({
+    mutationFn: async (newTodo) => {
```
Comment (DanDiac) [inline src/components/TodoContext.tsx:34]: `newTodo` parameter doesn't have a type defined

File: src/components/TodoContext.tsx [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+      });
+
+      if (!response.ok) {
+        throw new Error("Failed to add TODO");
+      }
+
+      const data = await response.json();
+      return data;
+    },
+    onSuccess: (data) => {
```
Comment (DanDiac) [inline src/components/TodoContext.tsx:50]: `data` parameter doesn't have a type defined
There are a  few more methods below that have parameters without a type defined.

Comment (StanCosmin28) [reply src/components/TodoContext.tsx:51]: done

**Comment by DanDiac** [comment]
You can merge.

**Pull request merged**